artifacts/deploy/admin-clusterrole-aggregation.yaml

```diff
@@ -0,0 +1,141 @@
+# This configuration is used to grant the admin clusterrole read
+# and write permissions for Karmada resources.
+
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRole
+metadata:
+  annotations:
+    rbac.authorization.kubernetes.io/autoupdate: "true"
+  labels:
+    kubernetes.io/bootstrapping: rbac-defaults
+    rbac.authorization.k8s.io/aggregate-to-view: "true"
+  name: system:karmada-view
+rules:
+  - apiGroups:
+      - "autoscaling.karmada.io"
+    resources:
+      - cronfederatedhpas
+      - cronfederatedhpas/status
+      - federatedhpas
+      - federatedhpas/status
+    verbs:
+      - get
+      - list
+      - watch
+  - apiGroups:
+      - "multicluster.x-k8s.io"
+    resources:
+      - serviceexports
+      - serviceexports/status
+      - serviceimports
+      - serviceimports/status
+    verbs:
+      - get
+      - list
+      - watch
+  - apiGroups:
+      - "networking.karmada.io"
+    resources:
+      - multiclusteringresses
+      - multiclusteringresses/status
+      - multiclusterservices
+      - multiclusterservices/status
+    verbs:
+      - get
+      - list
+      - watch
+  - apiGroups:
+      - "policy.karmada.io"
+    resources:
+      - overridepolicies
+      - propagationpolicies
+    verbs:
+      - get
+      - list
+      - watch
+  - apiGroups:
+      - "work.karmada.io"
+    resources:
+      - resourcebindings
+      - resourcebindings/status
+      - works
+      - works/status
+    verbs:
+      - get
+      - list
+      - watch
+
+---
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRole
+metadata:
+  annotations:
+    rbac.authorization.kubernetes.io/autoupdate: "true"
+  labels:
+    kubernetes.io/bootstrapping: rbac-defaults
+    rbac.authorization.k8s.io/aggregate-to-edit: "true"
+  name: system:karmada-edit
```

Comment:
I wonder why name the role with `system:` prefix.
> Caution: The prefix system: is reserved for Kubernetes system use, so you should ensure that you don't have users or groups with names that start with system: by accident.

Reply:
remove the `system:` prefix.